scripts/rabbitmq-server-ha.ocf

```diff
@@ -46,6 +45,7 @@ OCF_RESKEY_erlang_cookie_file_default="/var/lib/rabbitmq/.erlang.cookie"
 OCF_RESKEY_use_fqdn_default=false
 OCF_RESKEY_fqdn_prefix_default=""
 OCF_RESKEY_max_rabbitmqctl_timeouts_default=3
+OCF_RESKEY_policy_file_default="/usr/local/sbin/set_rabbitmq_policy.sh"
```

Comment:
this ^^ so , which path? or just OCF_RESKEY_policy_file_default="set_rabbitmq_policy" ?